opencti-platform/opencti-graphql/src/connector/importCsv/importCsv-connector.ts

```diff
@@ -56,42 +89,26 @@ const initImportCsvConnector = () => {
       const stream: SdkStream<Readable> | null | undefined = await downloadFile(fileId) as SdkStream<Readable> | null | undefined;
       await updateReceivedTime(context, applicantUser, workId, 'Connector ready to process the operation');
       if (stream) {
-        const chunks: Uint8Array[] = [];
-        let hasError: boolean = false;
-        stream.on('data', async (chunk) => {
-          chunks.push(chunk.toString('utf8'));
-        }).on('error', async (error) => {
-          hasError = true;
-          const errorData = { error: error.message, source: fileId };
-          await reportExpectation(context, applicantUser, workId, errorData);
-        }).on('end', async () => {
-          if (!hasError) {
-            const string = chunks.join('');
-            const bundle = await bundleProcess(context, applicantUser, Buffer.from(string), csvMapper, { entity });
-            const validateBeforeImport = connectorConfig.config.validate_before_import;
-            if (validateBeforeImport) {
-              await updateExpectationsNumber(context, applicantUser, workId, 1);
-              const contentStream = Readable.from([JSON.stringify(bundle, null, '  ')]);
-              const file = {
-                createReadStream: () => contentStream,
-                filename: `${workId}.json`,
-                mimetype: 'application/json',
-              };
-              await uploadToStorage(context, applicantUser, 'import/pending', file, { entity });
-              await reportExpectation(context, applicantUser, workId);
-            } else {
-              await updateExpectationsNumber(context, applicantUser, workId, bundle.objects.length);
-              const content = Buffer.from(JSON.stringify(bundle), 'utf-8').toString('base64');
-              await pushToSync({
-                type: 'bundle',
-                update: true,
-                applicant_id: applicantId ?? OPENCTI_SYSTEM_UUID,
-                work_id: workId,
-                content
-              });
+        let lines: string[] = [];
+        const rl = readline.createInterface({ input: stream, crlfDelay: 5000 });
```

Comment:
Is this 5000, a delay between 2 ingestions of chuncks ?

Reply:
No, its the delay for the streaming waiting for the end of line.